Android/app/src/main/java/realm/io/realmpop/util/RandomNumberUtils.java

```diff
@@ -14,7 +14,11 @@
 
         int[] numbers = new int[count];
         for(int i = 0; i < count; i++) {
-            numbers[i] = generateNumber(prevNum + 1, maxNum);
+            if (i == 0) {
```

Comment:
yup, that's the logic on iOS, helps the player know when they've tapped the last bubble